src/main/java/com/ericsson/eiffel/remrem/protocol/ValidationResult.java

```diff
@@ -1,21 +1,15 @@
 package com.ericsson.eiffel.remrem.protocol;
 
 public class ValidationResult {
-	
-	private boolean valid;
-	private String validationMesage;
-	
-	public boolean isValid() {
-		return valid;
-	}
-	public void setValid(boolean valid) {
-		this.valid = valid;
-	}
-	public String getValidationMesage() {
-		return validationMesage;
-	}
-	public void setValidationMesage(String validationMesage) {
-		this.validationMesage = validationMesage;
-	}
 
+    private boolean valid = true;
+    private String validationMesage;
+
+    public ValidationResult() {
+        super();
+    }
+    public ValidationResult(boolean valid, String validationMesage) {
+        this.valid = valid;
+        this.validationMesage = validationMesage;
+    }

```

Comment:
Great but getters are still needed otherwise subsequent operation cannot verify validity and the reason in message.